react/SearchContext.js

```diff
@@ -37,10 +37,10 @@ const SearchContext = ({
   children,
   __unstableProductOriginVtex,
 }) => {
-  const { page: runtimePage, query: runtimeQuery } = useRuntime()
+  const { page: runtimePage, query: runtimeQuery, route: { queryString: { map: renderMap } } } = useRuntime()
 
   const fieldsFromQueryString = {
-    mapField: runtimeQuery.map,
+    mapField: runtimeQuery.query ? runtimeQuery.map : renderMap,
```

Comment:
Why not always from route querystring?

Reply:
There's actually no need for it to be there, I've moved the map a bit further.